azure-devops/azext_devops/dev/repos/ref.py

```diff
@@ -70,6 +70,16 @@ def delete_ref(name, object_id, repository=None, organization=None, project=None
         project=project,
         repo=repository)
     client = get_git_client(organization)
+
+    if object_id is None:
+        ref = client.get_refs(repository_id=repository,
+                            project=project,
+                            filter=name)
+        if not ref:
```

Comment:
```suggestion
        if not ref or len(ref) > 1:
```

Reply:
just to be sure we are not getting multiple id's from service because in that case we can't be sure which one to delete